app/serializers/trainee_serializer/v01.rb

```diff
@@ -16,11 +22,24 @@ def as_hash
         hesa_trainee_attributes,
         nationality: nationality,
         training_initiative: training_initiative,
-        placements: @trainee.placements.map(&:attributes),
-        degrees: @trainee.degrees.map(&:attributes),
+        placements: placements,
+        degrees: degrees,
+        trainee_id: @trainee.slug,
```

Comment:
`trainee_id` is a field on the trainee which providers use to set their own internal identifiers in the UI. If they have set this, it feels weird we're overriding it to return our own slug. Might be worth getting some clarification on this

Reply:
do we have a different name for this? provider_trainee_id?

Reply:
no it's just `trainee_id` on the `Trainee` https://github.com/DFE-Digital/register-trainee-teachers/blob/main/db/schema.rb#L824. But it looks like `provider_trainee_id` is used in other tables to avoid confusion

Reply:
Perhaps we could just refer to the "internal" trainee_id as "external_trainee_id" within the api...

Reply:
Dave's gone for the ultimate solution, renaming the field for maximum clarity.